packages/Menu/README.md

```diff
@@ -0,0 +1,54 @@
+# @MESG-COMPONENTS/MENU
+
+## Install
+
+`npm install @mesg-components/menu` or `yarn add @mesg-components/menu`
+
+## Properties
+
+- **banner**: `String`, **Required**, -> _banner to display on the footer_
+- **items**: `Array of object`, **Optional**, -> _items to display on the footer_
+
+## Example
+
+```js
+import logo from '../assets/img/MESG-logo-horizontal-purple.svg'
+
+const categories = ['technology', 'developers', 'showcase', 'contributions', 'token', 'get-started']
+const items = [
+  {
+    key: 'technology',
```

Comment:
shouldn't be necessary anymore, same for the categories array

Reply:
done.